### ISSUE TITLE
Last line not shown in CDATA feeds

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kiniaru/mobile-rss-reader#36]
```
What steps will reproduce the problem?
1. mobile-rss-reader is not able to show the last line of every post in 
the following feed: http://bash.org.ru/rss The feed is in Russian and 
samle xml code is provided below.

What is the expected output? What do you see instead?
It's expected to show all (complete) entry and not to skip the last line.
All other available RSS readers (so far, PC- or mobile-based) are showing 
this feed correctly. 

What version of the product are you using? On what operating system?
1.11.1 and 1.11.2(development 3) on Sony C702

Please provide any additional information below.

Sample RSS/xml-code:

                <item>
                        <guid 
isPermaLink="false">95a0bc34c09bc0aee6408b6886129c
9114736b8df78551befc19925c07bed776</guid>
                        <link>http://bash.org.ru/quote/402401</link>
                        <title>entry #402401</title>
                        <pubDate>Thu, 05 Feb 2009 08:44:01 +0400</pubDate>
                        <description><![CDATA[text-line1<br>text-line2]]
></description>
                </item>
                <item>

In this example, only "text-line1" is displayed by mobile-rss-reader and 
all text after the last <br> is invisible.
This behaviour is only specific to this feed which is enclosing every 
entry in CDATA section.

Thanks.

```

Original issue reported on code.google.com by `mike.pet...@gmail.com` on 7 Feb 2009 at 8:17

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Thanks for reporting this and looking into it.  I will put a fix in for the 
next dev
release.

Regards,

Irving
```

Original comment by `ibunto...@gmail.com` on 9 Feb 2009 at 6:46

**Comment by GoogleCodeExporter** [comment]
```
Hi,

I have duplicated this problem.  Thanks again for your help.

Regards,

Irving
```

Original comment by `ibunto...@gmail.com` on 9 Feb 2009 at 10:17
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This problem is in the StringUtil.removeHtml method. When the HTML tags are 
removed,
the data after the last tag is ignored.  The result is that the whole data is 
not
displayed.  This will be put into the next dev release and next minor release 
1.11.2
```

Original comment by `ibunto...@gmail.com` on 10 Feb 2009 at 12:26
- Changed state: **Started**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
This is fixed on Beta 1 Version 1.11.2
```

Original comment by `ibunto...@gmail.com` on 16 Feb 2009 at 2:33

**Comment by GoogleCodeExporter** [comment]
```
Yes, it is fixed, thanks a lot!
You can close this issue.
```

Original comment by `mike.pet...@gmail.com` on 26 Feb 2009 at 7:08

**Comment by GoogleCodeExporter** [comment]
```
Thanks for verifying this.  I will close it once I release the final version.

Regards,

Irving
```

Original comment by `ibunto...@gmail.com` on 27 Feb 2009 at 6:09

**Comment by GoogleCodeExporter** [comment]
```
This is fixed on version 1.11.2.
```

Original comment by `ibunto...@gmail.com` on 14 May 2009 at 5:10
- Changed state: **Fixed**
- Added labels: **Milestone-Release1.11.2**, **OpSys-All**
